### PR TITLE
Bullet: bump version

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

File: sci-physics/bullet/bullet-2.85.1.recipe [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+	cmd:make
+	cmd:python
+	"
+
+BUILD()
+{
+	mkdir -p build_haiku
+	cd build_haiku
+
+	export LDFLAGS="-lroot -lnetwork -lbsd"
```
Comment (korli) [inline sci-physics/bullet/bullet-2.85.1.recipe]: -lroot can be dropped.

File: sci-physics/bullet/bullet-2.85.1.recipe [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	cmake .. \
+		-Wno-dev \
+		-DBUILD_SHARED_LIBS=ON \
+		-DINSTALL_LIBS=ON \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DBUILD_BULLET2_DEMOS=OFF \
+		-DBUILD_EXTRAS=OFF \
+		-DBUILD_UNIT_TESTS=OFF \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DLIB_SUFFIX="/x86/" \
```
Comment (korli) [inline sci-physics/bullet/bullet-2.85.1.recipe]: This looks wrong. Is it for x86 secondary arch?

Comment (korli) [inline sci-physics/bullet/bullet-2.85.1.recipe]: Please see https://github.com/haikuports/haikuports/commit/17a8d6fe2e0c7ee424ab1465707957484bc93116#diff-c749752d332bd50eceea802b7add7817R335

**Comment by scottmc** [comment]
Looks like this one just needs a little clean up?

**Comment by miqlas** [comment]
Can' test it on x86. I assume it is broken.

**Comment by miqlas** [comment]
I think one can merge it now.

**Pull request merged**